### PR TITLE
Add request form

### LINKED ISSUE #200: Create form to request access
Estimated duration: ~1.5d

See
#182 
https://github.com/geometalab/osmaxx/commit/c4817c37c49128393077fb8b2e16111c8231b795#commitcomment-13307969

Replace the user_manager_username by a group. Prompt this group assignment on creation of the superuser.

### PR DESCRIPTION
**Pull request by wasabideveloper**
Implementation of feature #200 (Merge will close #200 ).

* locales **NOT** compiled -> do this on release (prevent huge locale diffs in changes)
* ran application tests
* tested views by hand:
  * /access_denied
  * /request_access/
* tested successfully email sent by hand

To be reviewed by:
- [x] @das-g 
- [x] @hixi

### CODE REVIEW

File: osmaxx-py/osmaxx/excerptexport/urls.py [hunk 1]
```diff
@@ -8,7 +8,9 @@
     extraction_order_status,
     list_orders,
     access_denied,
-    order_form_view)
+    order_form_view,
+    request_access
```
Comment (hixi) [inline osmaxx-py/osmaxx/excerptexport/urls.py]: I'd like a trailing comma here ;-)

File: osmaxx-py/osmaxx/excerptexport/views.py [hunk 2]
```diff
@@ -136,16 +137,38 @@ def list_orders(request):
...
         )
 
-    view_context = {
-        'next_page': request.GET['next'],
-        'user': request.user,
-        'user_administrator_email': user_administrator_email,
-    }
-    return render_to_response('excerptexport/templates/access_denied.html', context=view_context,
-                              context_instance=RequestContext(request))
+    try:
```
Comment (hixi) [inline osmaxx-py/osmaxx/excerptexport/views.py]: I'd wrap only as much as needed with a try/except, eg.

(beware the incorrect indentation)

```python
email_message = _('{first_name} {last_name} requests access for Osmaxx. Please activate the user {username}.').format(
    username=request.user.username,
    first_name=request.user.first_name,
    last_name=request.user.last_name,
)
try:
    send_mail('Request access for Osmaxx', email_message, request.user.email,
        [user_administrator_email], fail_silently=True)
   messages.success(
           request,
           _('Your access request has been sent successfully. You will receive an email when your account is ready.')
       )
except...
```

File: osmaxx-py/osmaxx/excerptexport/views.py [hunk 3]
```diff
@@ -136,16 +137,38 @@ def list_orders(request):
...
+        )
+        send_mail('Request access for Osmaxx', email_message, request.user.email,
+                  [user_administrator_email], fail_silently=True)
+        messages.success(
+            request,
+            _('Your access request has been sent successfully. You will receive an email when your account is ready.')
+        )
+    except:
+        logging.exception(
+            "Sending access request e-mail failed! ("+email_message+")"
```
Comment (hixi) [inline osmaxx-py/osmaxx/excerptexport/views.py]: This won't help us much, we might want to have the message in the traceback, but the most crucial information is missing: what was the Exception, was it a connection failure, or a formatting issue, or whatnot...

Example, untested:

```python
except Exception as e:
    logging.exception("Sending access request e-mail failed: {0}, \n{1}".format(e, email_message)
```

File: osmaxx-py/osmaxx/excerptexport/urls.py [hunk 4]
```diff
@@ -20,14 +22,16 @@
 
     url(r'^orders/$', list_orders, name='orders'),
     url(r'^orders/new/$', order_form_view, name='new'),
-    url(r'^orders/(?P<extraction_order_id>[0-9]+)$', extraction_order_status, name='status')
+    url(r'^orders/(?P<extraction_order_id>[0-9]+)$', extraction_order_status, name='status'),
+
+    url(r'^request_access/$', request_access, name='request_access')
```
Comment (das-g) [inline osmaxx-py/osmaxx/excerptexport/urls.py]: Put a trailing comma here, too. (Even though we didn't have one previously.)

File: osmaxx-py/osmaxx/excerptexport/views.py [hunk 5]
```diff
@@ -136,16 +137,38 @@ def list_orders(request):
...
+        )
```
Comment (das-g) [inline osmaxx-py/osmaxx/excerptexport/views.py]: Shouldn't the _user_ get notified in this case, too? (Error page or via on-site message.)

File: osmaxx-py/osmaxx/excerptexport/urls.py (hunk 1 above)
Comment (wasabideveloper) [inline osmaxx-py/osmaxx/excerptexport/urls.py]: :heavy_check_mark:

File: osmaxx-py/osmaxx/excerptexport/views.py (hunk 2 above)
Comment (wasabideveloper) [inline osmaxx-py/osmaxx/excerptexport/views.py]: :heavy_check_mark:

File: osmaxx-py/osmaxx/excerptexport/views.py (hunk 3 above)
Comment (wasabideveloper) [inline osmaxx-py/osmaxx/excerptexport/views.py]: :heavy_check_mark:

File: osmaxx-py/osmaxx/excerptexport/urls.py (hunk 4 above)
Comment (wasabideveloper) [inline osmaxx-py/osmaxx/excerptexport/urls.py]: :heavy_check_mark:

File: osmaxx-py/osmaxx/excerptexport/views.py (hunk 5 above)
Comment (wasabideveloper) [inline osmaxx-py/osmaxx/excerptexport/views.py]: :heavy_check_mark:

**Pull request merged**